frontend/src/elements/UserAvatar.vue

```diff
@@ -7,15 +7,39 @@ const user = useUserStore();
 
 <template>
 <div
-  class="flex-center size-8 self-center rounded-full bg-white text-sm font-semibold text-white"
-  :class="{'!bg-teal-500': user.data.avatarUrl === null}"
+  class="user-avatar"
+  :class="{'user-avatar--fallback': user.data.avatarUrl === null}"
   data-testid="user-menu-avatar"
 >
   <span v-if="user.data.avatarUrl === null">
     {{ initials(user.data.name) }}
   </span>
   <span v-else>
-    <img class="size-[48px] rounded-full" :alt="initials(user.data.name)" :src="user.data.avatarUrl"/>
+    <img class="user-avatar__image" :alt="initials(user.data.name)" :src="user.data.avatarUrl"/>
   </span>
 </div>
 </template>
+
+<style scoped>
+.user-avatar {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 2rem;
+  height: 2rem;
+  align-self: center;
+  border-radius: 50%;
+  background-color: var(--colour-neutral-base);
+  font-size: 0.875rem;
+  font-weight: 600;
+  color: var(--colour-neutral-base);
+}
+
+.user-avatar--fallback {
```

Comment:
I don't mind the `__` convention. Can we switch the -- to __ here too?

Reply:
The style is also scoped so we probably drop the prefixes too. It'll be built as like user-avatar-<random hash>.